### PR TITLE
Update the README.md file to describe repo

### PR DESCRIPTION
**Pull request by AlexandraImbrisca**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,4 @@
-# Google Student Training in Engineering Program
-
-This repo contains the projects you'll work on for the first half of STEP.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
```
Comment (csperl) [inline README.md:1]: Why did you add a blank line at the beginning?

Comment (AlexandraImbrisca) [reply README.md:1]: I don't know actually, I think I put it by mistake. I will remove it now
